episodes/01-basics.md

```diff
@@ -62,6 +62,22 @@ metadata make up a [repository](../learners/reference.md#repository).
 Repositories can be kept in sync across different computers, facilitating
 collaboration among different people.
 
+In this lesson you will be using a version control system
+called [Git](learners/reference.md#git) alongside a
+cloud-based platform, [GitHub](learners/reference.md#github).
+By the end of the lesson you will have:
+
+- Configured Git based on our recommended settings
+- Initialised a new [repository](../learners/reference.md#repository)
```

Comment:
I'd say something along the lines of "Initialised a new repository with git" or "on the command line" since you can also do it through GitHub.